src/stlink-lib/sg.h

```diff
@@ -60,8 +57,5 @@ struct stlink_libsg {
 
 stlink_t* stlink_v1_open(const int verbose, int reset);
 
-#ifdef __cplusplus
-}
-#endif
 
```

Comment:
```suggestion
```

Reply:
Addressed in commit da3d9e3f1e1a6d116df5830cf506f30ff56196b3.